sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,4 @@
+select "c"."name" as "country", count("ci"."name") as "Number of cities"
```

Comment:
Although it works we typically want to avoid putting spaces in our aliased column names. This makes the data more difficult to work with in JavaScript.